jobs/pulse3d/pulse3d-worker/src/main.py

```diff
@@ -0,0 +1,84 @@
+import asyncio
+import json
+import logging
+import os
+import sys
+import tempfile
+
+import asyncpg
+import boto3
+
+from pulse3D.plate_recording import PlateRecording
+from pulse3D.excel_writer import write_xlsx
+from jobs import get_item, EmptyQueue
+
+logging.basicConfig(format='%(asctime)s %(message)s', level=logging.INFO, datefmt='%Y-%m-%d %H:%M:%S')
+logger = logging.getLogger(__name__)
+
+PULSE3D_UPLOADS_BUCKET = os.getenv(UPLOADS_BUCKET_ENV, "test-sdk-upload")
```

Comment:
does the env var key need to be a string?

Reply:
yep, my bad